docs/ref/models/expressions.txt

```diff
@@ -224,6 +224,12 @@ extra attribute ``field_lower`` produced, roughly, from the following SQL::
 
 See :doc:`database-functions` for a list of built-in database functions.
 
+.. versionchanged:: 1.9
```

Comment:
The annotation should contain an indented description of what changed. Generally this occurs after the main content.